watertap/data/techno_economic/component_list.yaml

```diff
@@ -82,8 +84,12 @@ perfluorootanoate_pfoa:
   constituent_longform: Perfluorootanoate (PFOA)
 phosphates:
   constituent_longform: Phosphates
+phosphorus:
```

Comment:
As a general question, is there sufficient difference in the definition of "phosphorous" from "phosphates" in this case?

@tarnold17 and I were discussing this in another PR, and decided to just go with phosphates there.

Reply:
I also had the same discussion with @tarnold17 and had encouraged him to only account for phosphates. However, I created a new yaml for secondary wastewater treatment and estimated average removal efficiencies based on a table in Metcalf & Eddy. Long story short-- I decided to stay true to the referenced table, which provides an entry for "total phosphorus".